css/header.css

```diff
@@ -0,0 +1,719 @@
+.header-wrapper--orange{
+    background-color: #fca53c;
+}
+
+.header-top{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    height: 50px;
+    color: #ffffff;
+}
+
+.header-top-text{
+    color: #ffffff;
+    text-decoration: none;
+    margin: 0 10px;
+}
+
+.header-top-left{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.header-top-right{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position: relative;
+}
+
+.header-top-right-button{
+    padding-left: 10px;
+    margin-right: -10px;
+    height: 50px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    box-sizing: border-box;
+}
+
+.header-top-right-list{
+    position: absolute;
+    padding: 10px 0 10px 30px;
+    width: 200px;
+    margin: 0px;
+    color: #DFDFDF;
+    background-color: #ffffff;
+    top:60px;
+    right: 0px;
+    z-index: 1;
+    box-shadow: 0px 0px 7px 0px #858585;
+    opacity: 0;
+    visibility: hidden;
+    transition: visibility .5s, opacity .5s, top .5s;
+}
+
+.header-top-right-list li{
+    padding: 10px;
+}
+
+.header-top-right-list-text{
+    color: black;
+    padding: 10px 0;
+    text-decoration: none;
+}
+
+.header-top-right-button:hover .header-top-right-list,
+.header-top-right-list:hover{
+    opacity: 1;
+    top: 50px;
+    visibility: visible;
+    transition: opacity .5s, top .5s;
+}
+
+.header-top-right-list-text:hover{
+    color: #fca53c;
+}
+
+.header-mid{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    flex-wrap: wrap;
+}
+
+.header-mid-logo{
+    width: 300px;
+    display: flex;
+    justify-content: space-around;
+    padding: 20px 0;
+}
+
+.header-mid-right-search-list{
+    display: flex;
+}
+
+.header-mid-right-search-input{
+    color: rgb(131, 127, 127);
```

Comment:
could you replace it with hex please? https://www.hexcolortool.com/

Reply:
corrected